### PR TITLE
[Doc, deeper look] acquire public key cert needs example, problems re…

### PR DESCRIPTION
**Pull request by arifBurakDemiray**
…work

### CODE REVIEW

**Comment by arifBurakDemiray** [comment]
[[Doc, deeper look] acquire public key cert needs example, problems rework](https://trello.com/c/LLSaUToG/1042-doc-deeper-look-acquire-public-key-cert-needs-example-problems-rework)

File: a_deeper_look_at_sdk_concepts/current.md [hunk 1]
```diff
@@ -1010,24 +1010,121 @@ module.exports = {
...
+    PSK identity: None
+    PSK identity hint: None
+    SRP username: None
+    Start Time: 1642277969
+    Timeout   : 7200 (sec)
+    Verify return code: 0 (ok)
+---
+</code></pre>
+<p>
+  <span>The produced output would contain information about multiple certificates, they are showing the certificate chain of trust to the respective root certificate authority. The specific cert information for your server would be at the top and that is the one you would need to copy.</span>
```
Comment (turtledreams) [inline a_deeper_look_at_sdk_concepts/current.md:1110]: tone should consistent. "they are showing" sounds off. Maybe "it shows"?

File: a_deeper_look_at_sdk_concepts/current.md [hunk 2]
```diff
@@ -1010,24 +1010,121 @@ module.exports = {
...
+</p>
+<p>
+  <span>You would copy the string/bytes betwrrn the first <code lang="bash">---BEGIN CERTIFICATE---</code> and <code lang="bash">-----END CERTIFICATE-----</code> tags and paste them to init block of the SDK that you are using. Remember to not add any newlines when providing this to the SDK.</span>
+</p>
+<h1 id="h_01HDNJK8PAE5GEQWRFDS4KD6S6">Common SSL certificate problems</h1>
+<p>
+  <span>Problems might be encountered related to SSL or certificate exceptions. <a href="https://developer.android.com/privacy-and-security/security-ssl">Here</a> is a list of common reasons for issues in Android.</span><span></span>
+</p>
+<p>
+  To investigate those issues further, sometimes a good way of exploring the cause
```
Comment (turtledreams) [inline a_deeper_look_at_sdk_concepts/current.md:1120]: looks tautological. I think omitting "To investigate those issues further" would do

File: a_deeper_look_at_sdk_concepts/current.md [hunk 3]
```diff
@@ -1010,24 +1010,121 @@ module.exports = {
...
+  of the problem is the same openssl certificate command:
 </p>
 <pre><code lang="bash">openssl s_client -connect xxx.server.ly:443 -showcerts</code></pre>
-<p>It's error codes can sometimes lead you to a solution.</p>
+<p>Example output might look like this:</p>
+<pre><code lang="bash">0056931101000000:error:10080002:BIO routines:BIO_lookup_ex:system lib:crypto/bio/bio_addr.c:738:nodename nor servname provided, or not known
+connect:errno=0</code></pre>
+<p>It's error codes may lead to a solution.</p>
```
Comment (turtledreams) [inline a_deeper_look_at_sdk_concepts/current.md:1127]: its
it's => it is(has)
its => belongs to 'it'
or you can go for "the error codes here ..."

**Review by turtledreams — CHANGES_REQUESTED** [review]

**Review by ArtursKadikis — APPROVED** [review]

**Pull request merged**